packages/flame/lib/src/game/game.dart

```diff
@@ -341,7 +344,7 @@ abstract mixin class Game {
 
   set mouseCursor(MouseCursor value) {
     _mouseCursor = value;
-    refreshWidget();
+    refreshWidget(internalRefresh: false);
```

Comment:
This one is the same as adding a gesture detector so this should be true

Reply:
Implemented, although I looked at the code and it doesn't implement the injection of a widget the others do, but I can see how its going to attach the cursor to the nearest node in the tree, so I presume the event lifecycle becomes the same and thus we would not want attach / detach to occur.